src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -6981,7 +6986,9 @@ namespace OutputReportTabular {
 
 			// get change in overall state of charge for electrical storage devices.
 			if ( NumElecStorageDevices > 0 ) {
-				OverallNetEnergyFromStorage = ( sum( ElecStorage.StartingEnergyStored() ) - sum( ElecStorage.ThisTimeStepStateOfCharge() ) );
```

Comment:
@Myoldmopar Thanks for finding that!  Because ElecStorage was removed from this line, the using ManageElectricPower::ElecStorage; is no longer needed in this routine.  Subtle.